modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -8,6 +8,8 @@ You can use multiple client providers, to help you enforce security and regulati
 
 You can use an Anypoint Platform native client provider (default) or configure an external client provider. To assign separate client providers for different organizations and environments, you must first enable client applications to be authorized using OAuth for the client providers that you want to implement in your organization.
 
+In previous versions of Anypoint Platform, only a single IdP: either an external IdP or the default IdP (Anypoint Platform native client provider) was supported. However, with the multiple client providers feature, Anypoint Platform supports either one default IdP, or one or more external IdPs.
```

Comment:
To make this text future-proof, consider naming the precise release number:
```suggestion
In Anypoint Platform [x.x] and earlier, only a single IdP was supported: either an external IdP or the default IdP (Anypoint Platform native client provider). With the addition of the multiple client providers feature, Anypoint Platform [x.x] and later supports either one default IdP or one or more external IdPs.
```

Reply:
Done! Thanks!